srcts/extras/busy-indicators/busy-indicators.scss

```diff
@@ -46,6 +50,59 @@
   }
 }
 
+@mixin spinner-svg($name) {
+  [data-shiny-busy-spinners="#{$name}"] {
+    --_shiny-spinner-url: url("spinners/#{$name}.svg");
+  }
+}
```

Comment:
I previously had the customization of spinner types working entirely off of CSS variables -- I think I'd prefer to go back to that https://github.com/rstudio/shiny/blob/c40f9f1de8ef7f30ca89cd0fe22c6e587c9929b4/srcts/extras/busy-indicators/busy-indicators.scss

Reply:
Agreed. I did something similar but it was made a little simpler by moving the spinners into the `busyIndicatorDependency()` files.